MAUI/DataGrid/export-to-pdf.md

```diff
@@ -0,0 +1,665 @@
+
+## Row height and column width customization
+
+### ExportColumnWidth
+
+By default, the data grid columns will be exported to PDF with `DataGridPdfExportingOption.DefaultColumnWidth` value. To export the data grid to PDF with exact column widths, set the `DataGridPdfExportingOption.CanExportColumnWidth` to `true`.
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.CanExportColumnWidth = true;
+{% endhighlight %}
+{% endtabs %}
+
+### ExportRowHeight
+
+By default, the data grid rows will be exported to PDF with `DataGridPdfExportingOption.DefaultRowHeight` value. To export the data grid to PDF with exact row heights, set the `DataGridPdfExportingOption.CanExportRowHeight` to `true`. 
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.CanExportRowHeight = true;
+{% endhighlight %}
+{% endtabs %}
+
+### DefaultColumnWidth
+
+The SfDataGrid allows customizing column width in the PDF document using the `DataGridPdfExportingOption.DefaultColumnWidth` property. The `DefaultColumnWidth` value will be applied to all the columns in the document.
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.DefaultColumnWidth = 150;
+option.CanExportColumnWidth = false;
+{% endhighlight %}
+{% endtabs %}
+
+### DefaultRowHeight
+
+The SfDataGrid allows customizing row height in the PDF document using the `DataGridPdfExportingOption.DefaultRowHeight` property. The `DefaultRowHeight` value will be applied to all the rows in the document.
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.DefaultRowHeight = 80;
+option.CanExportRowHeight = true;
+{% endhighlight %}
+{% endtabs %}
+
+## Events
+
+The SfDataGrid provides you the following events for exporting:
+
+* `RowExporting`: Raised when exporting a row at the execution time.
+* `CellExporting`: Raised when exporting a cell at the execution time.
+
+### RowExporting
+
+The `DataGridRowPdfExportingEventHandler` delegate allows customizing the styles of record rows and group caption rows. The `RowExporting` event is triggered with `DataGridRowPdfExportingEventArgs` that contains the following properties:
+
+* `PdfGrid`: Customizes the pdfGrid properties such as `Background`, `CellPadding`, `CellSpacing`, and so on.
+* `PdfRow`: Specifies the `PDFGridRow` to be exported. Customizes the properties of a particular row. 
+* `Record`: Gets the collection of exported underlying data objects.
+* `RowType`: Specifies the row type using `ExportRowType` enum. Checks the row type and applies different styles based on the row type.
+
+You can use this event to customize the properties of the grid rows exported to PDF. The following code example illustrates how to change the background color of the record rows and caption summary rows when exporting.
```

Comment:
Grammar check and revised content:

The `DataGridRowPdfExportingEventHandler` delegate allows for customizing the styles of record rows and group caption rows. The `RowExporting` event is triggered with `DataGridRowPdfExportingEventArgs`, which contains the following properties:

* `PdfGrid`: Customizes the properties of the `pdfGrid`, such as `Background`, `CellPadding`, `CellSpacing`, and more.
* `PdfRow`: Specifies the `PDFGridRow` to be exported and allows customization of the properties of a particular row.
* `Record`: Retrieves the collection of exported underlying data objects.
* `RowType`: Specifies the row type using the `ExportRowType` enum. It checks the row type and applies different styles based on the type.

You can use this event to customize the properties of the grid rows that are exported to PDF. The following code example demonstrates how to change the background color of the record rows and caption summary rows during the export process.

Reply:
Updated the content